ts/Core/Scrollbar.ts

```diff
@@ -661,8 +665,8 @@ class Scrollbar {
             size = scroller.size,
             styledMode = scroller.chart.styledMode,
             group = renderer.g('scrollbar').attr({
-                zIndex: options.zIndex,
-                translateY: -99999
+                visibility: 'hidden',
```

Comment:
In this case too it will probably be better to use the shorthand `.hide()` function. It is chainable.